express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+const grades = {};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const gradeArr = [];
+  for (const grade in grades) {
+    gradeArr.push(grades[grade]);
+  }
+  res.status(201).json(gradeArr);
+});
+
+app.use(express.json());
+
+let nextId = 1;
```

Comment:
I'd put this up at the top, too. It's good to have all your global variables at the top.